docs/INSTALLATION.md

```diff
@@ -0,0 +1,7 @@
+# Install Dyalog
+You can get started with APL and even tackle some of the exercises without installing anything on [TryAPL.org](https://tryapl.org). However, in order to run the test cases you must install the Dyalog interpreter on your machine.
+
+Dyalog is available on Microsoft Windows, macOS and Linux. There is also a public Docker container allowing you to run Dyalog on any platform with Docker available. Go to [https://www.dyalog.com/download-zone.htm](https://www.dyalog.com/download-zone.htm) to Download Dyalog.
+
+> **NOTE:**
+  On Linux, we recommand also installing the Ride graphical IDE. More information and installation instructions can be found in [the Ride documentation](https://dyalog.github.io/ride).
```

Comment:
@rikedyp FWIW, I'm running RIDE on Mac just fine

Reply:
Yes but on mac Dyalog comes with Ride out-of-the-box. On Linux you must install Ride separately.